src/client/components/VotingBox.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const VotingBox = ({ voteForItem, votesNumber, whetherUserVoted }) => (
+  <button
+    aria-label="Vote for item"
+    className={classNames('voting-box', {
+      'voting-box__voted': whetherUserVoted
+    })}
+    type="button"
+    onClick={e => {
+      e.stopPropagation();
+      voteForItem();
+    }}
+  >
+    <i className="fas fa-thumbs-up voting-box__icon" />
```

Comment:
consider using local SVG file, import it and place as `<VotingIcon />`
I did something like this in SortBox component - check it out.

Reply:
Changed to local svg. In order not to modify the color of the image, I changed the button styles gently.